src/spyglass/position/v1/position_dlc_selection.py

```diff
@@ -350,10 +350,23 @@ def make(self, key):
             },
             populate_missing=False,
         )
-        epoch = (
-            int(interval_name.replace("pos ", "").replace(" valid times", ""))
-            + 1
-        )
+
+        if isinstance(interval_name, str):
```

Comment:
it seems like the only use of `interval_name` in this function is to get the epoch (which is already in the key). could we instead skip this and just assign `epoch = key["epoch"]`?

Reply:
My thought process was that the returned interval name might not exactly match the key fed to `convert_epoch...`. If it always will, I'm not sure what purpose the function serves

Reply:
But epoch is already in the primary key of this table so we don't need to to any of the name-based lookup steps.  Checking the history, I had cut this out in #975, but I think it didn't get merged correctly with later #870

Reply:
Ah, good catch